### PR TITLE
Move information to the sidebar, enforce max width

### PR DESCRIPTION
**Pull request by zbynek**
This PR
* moves some elements from the header to the sidebar
* sets max-width (and slightly increases the font size) to make sure the documentation easier to read
* changes wording in a few places to make text more concise
* makes the graph smaller and removes y-axis labels; together with the current number of installs this should provide enough info (the values are still available on mouseover).
* removes a duplicated GitHub link

image

Most of these changes are inspired by NPM.

### CODE REVIEW

File: src/components/PluginReadableInstalls.jsx
Comment (halkeye) [inline src/components/PluginReadableInstalls.jsx:8]: I've never used this util. If you specify the format does it only do it for us style numbers? Is possible to let the browser pick the locale?

**Review by halkeye — APPROVED** [review]
Lgtm.

**Review by timja — APPROVED** [review]

File: src/components/PluginReadableInstalls.jsx
Comment (zbynek) [reply src/components/PluginReadableInstalls.jsx:8]: If we omit the locale name, the browser locale will be used. But when formatting dates we are also using US locale and [almost](https://plugins.jenkins.io/redis-notifier/) all the texts are in English, so I thought this is more consistent.

Comment (halkeye) [reply src/components/PluginReadableInstalls.jsx:8]: Oh yea totally a good plan. I was just curious how it worked.

I think I started separating dates into its own components for that reason, so we could eventually fix it. I just didn't go far enough

File: src/components/LineChart.jsx
Comment (daniel-beck) [inline src/components/LineChart.jsx]: This can be done much more simply if we're getting rid of the Y axis labels, as the only thing this seems to accomplish having no weird (non-even, differently spaces) outlier as the max Y value.

Just use `maxValue*1.2` or so as the `max`?

**Review by daniel-beck — APPROVED** [review]
Looks reasonable. Especially de-emphasizing the (often outdated) maintainers list is a good idea.

The graph takes some getting used to, but I can see it being there to only show the trend, and it should be fine for that. Glad we got the y=0 integrated before this 😉 

Curious why we're lot putting all the metadata to the side, in a "Current Release" section? (Except for ID, which applies to all releases. That one may be best above the tabs, next to the label.)

**Comment by daniel-beck** [comment]
Thanks for the review request @timja !

**Comment by zbynek** [comment]
> Curious why we're not putting all the metadata to the side, in a "Current Release" section? (Except for ID, which applies to all releases.)

I didn's see a strong reason to move those and [NPM](https://www.npmjs.com/package/jenkins) is also not putting this information in the sidebar. But it can be moved of course. If we'd that, where should the ID go?

**Comment by daniel-beck** [comment]
> But it can be moved of course. If we'd that, where should the ID go?

My suggestion would be to try below display name and above tabs.

**Comment by zbynek** [comment]
@daniel-beck I moved the version to the right; the ID is in a floating block that's either below the title or to the right, depending on ID length and screen size.

> in a "Current Release" section

Currently the version number itself is used as a heading to some space.

**Review by daniel-beck — APPROVED** [review]
This looks great 😻

File: src/components/LineChart.jsx
Comment (zbynek) [reply src/components/LineChart.jsx]: Fixed in the last commit.

**Comment by zbynek** [comment]
Given the approvals I'll merge this in 24 hours if there is no negative feedback.

**Review by daniel-beck — APPROVED** [review]
Re-:+1:

**Pull request merged**
